packages/react-dom/src/__tests__/ReactDOMServerPartialHydration-test.internal.js

```diff
@@ -1888,7 +1888,7 @@ describe('ReactDOMServerPartialHydration', () => {
     const TestResponder = React.DEPRECATED_createResponder(
       'TestEventResponder',
       {
-        targetEventTypes: ['click'],
```

Comment:
This test didn't catch it because apparently this means that it's listening to *both* active and passive. With the fix, now it actually fires twice which I think is a bug.